spellbook/models/magiceden/solana/magiceden_solana_trades.sql

```diff
@@ -0,0 +1,33 @@
+ {{
+  config(
+        alias='trades',
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key='unique_id'
+  )
+}}
+
+SELECT 
+  signatures[0] || id as unique_id,
+  'solana' as blockchain,
+  signatures[0] as tx_hash, 
+  block_time,
+  abs(post_balances[0] / 1e9 - pre_balances[0] / 1e9) * p.price AS amount_usd,
+  abs(post_balances[0] / 1e9 - pre_balances[0] / 1e9) AS amount,
+  p.symbol as token_symbol,
+  p.contract_address as token_address,
+  account_keys[0] as traders,
+  id as trade_id
+FROM {{ source('solana','transactions') }}
+LEFT JOIN {{ source('prices', 'usd') }} p 
+  ON p.minute = date_trunc('minute', block_time)
+  AND p.symbol = 'SOL'        
+WHERE (array_contains(account_keys, 'MEisE1HzehtrDpAAT8PnLHjpSSkRYakotTuJRPjTpo8') -- magic eden v1
```

Comment:
I think what Jillian was alluding to during our call was that things like this account reference could be macros. so instead of copy pasting this address we could reference it once in a macro.